File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        count={}
        for i in nums:
            count[i] = 1 + count.get(i, 0)
        for k,v in count.items():
            if v>(len(nums))/2:
                return k
        

File: Data Structures & Algorithms/binary-search/submission-0.cpp
class Solution {
public:
    int search(vector<int>& nums, int target) {
        long long n;
        n=nums.size();
        long long left=0;
        long long right=n-1;
        while(left<=right){
            long long mid=(left+right)/2;
            if(nums[mid]==target){
                return mid;
            }
            else if (nums[mid]<target){
                left=mid+1;
            }
            else{
                right=mid-1;
            }
        }
        return -1;
    }
};
